docs/designs/2848-metapackage-proposal.md

```diff
@@ -0,0 +1,169 @@
+# Metapackage-based Installation Experience
+
+* Proposal: https://github.com/vmware-tanzu/community-edition/issues/2848
+
+## Summary
+
+In order to simplify the experience of installing multiple packages, we wanted to leverage the ability of creating a "metapackage" using Carvel tooling. 
+
+This metapackage is essentially a package of packages wherein it references a Bill of Materials that includes existing packages in the repository. Being able to install, update & remove these packages in concert with each other simplifies the complexity significantly. 
+
+This is a method in practice in other projects leveraging Carvel tooling. 
+
+## Motivation
+
+The motivation for introducing this is to reduce the complexity that comes up with installing multiple carvel packages at a time. This will allow newcomers to the project rapidly establish a working technology stack.
+
+* Describe why the change is important and the benefits to users.  
+
+### Goals
+
+* Demonstrate the ability that multiple subordinate packages can be installed using a single metapackage. 
+* Be able to provide a single values.yaml file that can be used for all the subordinate package values.yaml.
+
+### Non-Goals/Future Work
+
+* Define each profile (and the packages it will have) that will use this metapackage technology. 
+
+## Proposal
+
+The proposal will describe what a metapackage is and how it will be implemented. 
+
+The proposal can be leveraged for various user scenarios. One example user scenario is described below. 
+
+**Terminology**
+
+* Metapackage - concept to bring two or more packages together. 
+* Profile - Enabling users to decide by a config value which part of the metapackage the user wants to install. 
+
+
+
+### Example Scenario
+
+The below scenario is one use-case of the metapackage technology. This is **not** the only way to use it. 
+
+
+- foo metapackage
+- bar1, bar2 and bar3 are the three subordinate packages part of foo. 
+- **lite** and **full** are the two profiles we will use
+
+
+1. Listing all the packages  
+
+`tanzu package available list`
+
+```
+NAME                    DISPLAY-NAME          SHORT-DESCRIPTION
+  
+foo.tanzu.vmware.com    Foo                   Used to explain metapackages
+bar1.tanzu.vmware.com   Bar1                  First subordinate package
+bar2.tanzu.vmware.com   Bar2                  Second subordinate package
+bar3.tanzu.vmware.com   Bar3                  Third subordinate package
+```
+
+The subordinate packages as part of each of the profiles. 
+
+| lite  | full  |
+| ----- | ----- |
+| bar 1 | bar 1 |
+| bar 2 | bar 2 |
+|       | bar 3 |
+
+2. Installing a profile
+* List version information for the package by running:
+
+```
+tanzu package available list foo.tanzu.vmware.com 
+```
+* Create a `foo-values.yaml` file. This configuration file will have configuration required to deploy foo and the subordinate packages. 
```

Comment:
I don't think this should be a single yaml file: these could end up being huge and I think breaking them apart will make them easier for users to grok.

For example, lets say there was some observability metapackage that was Grafana, Prometheues, and Fluent. That could easily run 1000 lines of yaml if it was all in one.